### ISSUE TITLE
[FEATURE]: show something when something in a command(or part of it) fails

### ISSUE THREAD

**Issue by kroher** [garden-io/garden#5256]
## Feature Request

For garden 0.13,  if something in a `garden deploy --dev` command fails, i dont see anything showing thats the case.  Right now it just shows "Exited" or "Watching" instead of some indicator about what or how many things failed.  Would be super useful to find past failures and evaluate issues over time that users are hitting where right now i'd have to just drill into every command to find the one i'm looking for.

🌹 It’s a nice to have, but nice things are nice 🙂

**Comment by stefreak** [comment]
@kroher Thank you for reporting this issue! Can you provide a reproducable example for your particular issue, together with actual behaviour and the behaviour you expected?

If something fails, we strive display error messages, so if we do not do that in your particular case that would be considered a bug actually.

**Comment by stefreak** [comment]
Reading this again, is this a feature request for Garden Cloud @kroher?

**Comment by kroher** [comment]
yeah garden cloud for commands run using bonsai (didnt try acorn).  the super simple example is making a container module that builds a docker image with a super low timeout.  that action will have failed pieces in the detail, but i don't have any indication in the command list without drilling into each row (i only see exited or watching).  it would be good to see something related to the actions(error/running/successful) in there though the errors are usually what i care about.

![image](https://github.com/garden-io/garden/assets/139928/f3996e8d-2780-43be-b248-c46e4aeea4a4) 
![image](https://github.com/garden-io/garden/assets/139928/8084ec00-f664-4eec-b889-7d1cabe45bd1)
![image](https://github.com/garden-io/garden/assets/139928/63f06a21-c03d-47ec-a17d-452d1a428f94)
